src/drawing/DrawingAnnotations.tsx

```diff
@@ -6,6 +6,7 @@ import DrawingCreator from './DrawingCreator';
 import DrawingPathGroup from './DrawingPathGroup';
 import DrawingSVG, { DrawingSVGRef } from './DrawingSVG';
 import DrawingSVGGroup from './DrawingSVGGroup';
+import PopupBase from '../components/Popups/PopupBase';
```

Comment:
We should export an alias of the ref type from `PopupDrawingToolbar` to avoid having to import the base component here. Several of our other components follow this model.

Reply:
Will update.